FirstApp/ViewControllers/HomeViewController.swift

```diff
@@ -1,7 +1,50 @@
 import UIKit
 
 class HomeViewController: UIViewController {
+
+    var name = ["mon4", "mon5", "mon6", "mon7", "mon8", "mon9", "mon10", "cherry", "apples"]
+
+    @IBOutlet weak var tableView: UITableView!
+
+    @IBOutlet weak var recipeSearchBar: UISearchBar!
+
+    var foodList = [FoodDetails]() {
+    didSet {
+               DispatchQueue.main.async {
+                   self.tableView.reloadData()
+               }
+           }
+       }
     override func viewDidLoad() {
         super.viewDidLoad()
+recipeSearchBar.delegate = self
```

Comment:
indenting is weird

Reply:
Thanks for noticing. I have now indented everything.